utils/utils.py

```diff
@@ -376,3 +377,11 @@ async def get_file_from_url(
     return discord.File(
         BytesIO(response), filename=filename
     )  # convert response to files
+
+
+def is_user_banned_openAI(user: Identifiable, /):
```

Comment:
Snake case for function names

Reply:
Also IMO make this a static/class method of OpenAIAdmin, or at least put in the same file, since this function is related to that only too.